Autopilot/AttitudeManager/AttitudeManager.h

```diff
@@ -96,7 +99,7 @@
 #define FLAP_IN_CHANNEL 5
 #define FLAP_OUT_CHANNEL 5
 #define UHF_STATUS_IN_CHANNEL 7
-#define PROBE1_OUT_CHANNEL 6
+#define PROBE1_OUT_CHANNEL 6 //TODO: Remove the probe drop channels as they are no longer used
```

Comment:
I'll probably move these typedefs out into PWM.h since it makes more sense. We've got too much defined in AttitudeManager.h. I'll do this in a seperate PR